Packs/PrismaSaasSecurity/ModelingRules/SaasSecurityEventCollector.xif

```diff
@@ -0,0 +1,16 @@
+[MODEL: dataset=Palo_Alto_Networks_Saas_Security_raw, model=Audit]
```

Comment:
We need to test this works before release. We dont need one for each type we pull from saas security?

Reply:
i thought the same as you at first, but i talked with Elia from security team and he explained to me this is how it should have been done. Obviously it will be tested before release.